_data-prepper/pipelines/configuration/sources/dynamo-db.md

```diff
@@ -0,0 +1,93 @@
+2.  Stream events from DynamoDB streams without full initial snapshots. This is useful for users who already have a mechanism for snapshots within their pipelines. This requires that DynamoDB stream is enabled on the DynamoDB table.
+
+## Usage
+
+The following example pipeline specifies DynamoDB as a source. It ingests data from a DyanmoDB table, named `table-a` through a PITR snapshot. It also indicates the `start_position`, which tells the pipeline how to read events DynamoDB stream. 
+
+```yaml
+version: "2"
+cdc-pipeline:
+  source:
+    dynamodb:
+      tables:
+        - table_arn: "arn:aws:dynamodb:us-west-2:123456789012:table/table-a"
+          export:
+            s3_bucket: "test-bucket"
+            s3_prefix: "myprefix"
+          stream:
+            start_position: "LATEST" # Read latest data from streams (Default)
+      aws:
+        region: "us-west-2"
+        sts_role_arn: "arn:aws:iam::123456789012:role/my-iam-role"
+```
+
+## Configuration options
+
+The following tables describe the configuration options for the `dynamodb` source.
+
+
+The following settings are shared between the `dynamodb` source.
+
+Option | Required | Type | Description
+:--- | :--- | :--- | :---
+`aws` | Yes | AWS | The AWS configuration. See [aws](#aws) for more information.
+`acknowledgments` | No | Boolean  | When `true`, enables `s3` sources to receive [end-to-end acknowledgments]({{site.url}}{{site.baseurl}}/data-prepper/pipelines/pipelines#end-to-end-acknowledgments) when events are received by OpenSearch sinks.
+`shared_acknowledgement_timeout` | No | Duration | The time before the data read from a DynamoDB stream expires when used with acknowledgements. Default is 10 minutes.
+`s3_data_file_acknowledgment_timeout` | No | Duration | The time before the data read from a DynamoDB export expires when used with acknowledgments. Default is 5 minutes.
+`tables` | Yes | List | The configuration for the DynamoDB table. See [tables](#tables) for more information.
+
+### aws
+
+Use the following options in the AWS configuration:
+
+Option | Required | Type | Description
+:--- | :--- | :--- | :---
+`region` | No | String | The AWS Region to use for credentials. Defaults to [standard SDK behavior to determine the Region](https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/region-selection.html).
+`sts_role_arn` | No | String | The AWS Security Token Service (AWS STS) role to assume for requests to Amazon SQS and Amazon S3. Defaults to `null`, which will use the [standard SDK behavior for credentials](https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/credentials.html).
+`aws_sts_header_overrides` | No | Map | A map of header overrides that the IAM role assumes for the sink plugin.
+
+
+### tables
+
+Use the following options with the `tables` configuration.
```

Comment:
I think we should add this table here. This way, users know what options come next.

```
Option | Required | Type | Description
:--- | :--- | :--- | :---
`table_arn` | Yes | String | The ARN of the DynamoDB table to use as a source.
`export` | No | Export | See the [export](#export) options.
`stream` | No | Stream | See the [stream](#stream) options.
```